src/utils/get-sorted-nodes-by-import-order.ts

```diff
@@ -14,12 +19,19 @@ import {
  * @param originalNodes A subset (of all import nodes) that should be sorted.
  * @param options Options to influence the behavior of the sorting algorithm.
  */
-export const getSortedNodesByImportOrder: GetSortedNodes = (
+export const getSortedNodesByImportOrder: GetSortedNodesByImportOrder = (
     originalNodes,
-    options,
+    { importOrder },
 ) => {
-    // This normalization is safe even if the option is already correct.
-    const importOrder = normalizeImportOrderOption(options.importOrder);
+    if (
+        process.env.NODE_ENV === 'test' &&
+        JSON.stringify(importOrder) !==
+            JSON.stringify(testingOnly.normalizeImportOrderOption(importOrder))
+    ) {
+        throw new Error(
+            'API Misuse: getSortedNodesByImportOrder::importOrder option already should be normalized.',
+        );
+    }
```

Comment:
If we think this could be a point of confusion, maybe it's worth renaming the option after we process it?  Then we could rely on TypeScript vs runtime (testtime) checks.

Reply:
This is only matters when people are directly calling this utility function, eg. From our spec files?

I threw this error here and then was able to track down all the test-files that needed corrections for moving the importOrder normalization to somewhere else.

I’ve tagged this if-block so it only executes in `test` but if you prefer, we could delete it.